maven/maven.bzl

```diff
@@ -179,11 +199,21 @@ def _generate_maven_repository_impl(ctx):
         content = "\n".join([prefix] + target_definitions)
         ctx.file(file, content)
     if bool(missing_artifacts):
```

Comment:
I would just move all this logic (inside `if bool(missing_artifacts):` condition) to its own little function responsible for printing nicely formatted message.

Reply:
Yeah, I was thinking of that.  I'll do it.